src/fhirtypes/ElementDefinition.ts

```diff
@@ -143,6 +145,17 @@ export class ElementDefinition {
     return this.path.slice(this.structDef.type.length + 1);
   }
 
+  /**
+   * Get the StructureDefinition for ElementDefinition
+   * @param {ResolveFn} resolve - A function that can resolve a type to a StructureDefinition instance
+   */
+  getOwnStructureDefinition(resolve: ResolveFn = () => undefined) {
```

Comment:
Can you add a return type annotation so it's clear to users (instead of inferred)?  This also ensures if you ever change the function, you don't unintentionally change the return type.